### DISCUSSION TITLE
Pallas flash attention benchmarks

### DISCUSSION THREAD

**Discussion by jenkspt** [jax-ml/jax#18314] [Q&A]
I'm very excited to see the new documentation for [Pallas](https://jax.readthedocs.io/en/latest/pallas/index.html). I'm also excited to see a flash attention implementation. I'm wondering if there are any TPU / GPU benchmarks for the pallas flash attention kernel?

Related to #18121

**Comment by MoFHeka** [comment]
Any update？
